vocode/streaming/streaming_conversation.py

```diff
@@ -141,8 +142,12 @@ async def start(self, mark_ready: Optional[Callable[[], Awaitable[None]]] = None
         if self.synthesizer.get_synthesizer_config().sentiment_config:
             self.update_bot_sentiment()
         if self.agent.get_agent_config().initial_message:
-            self.send_message_to_stream_nonblocking(
-                self.agent.get_agent_config().initial_message, False
+            self.agent_responses_queue.put_nowait(
+                (
+                    self.agent.get_agent_config().initial_message,
+                    False,
+                    self.enqueue_stop_event(),
```

Comment:
I'm not too sure about continuing the pattern of enqueue_stop_event. I find it hard to properly grasp and I'm not sure why it's necessary.

As far as I understand, as soon as a human speaks (as detected by the handle_transcription_task), you want to interrupt (in that order but in quick succession): 
- the playback
- the synthesizer
- the llm
The best way to do that is to get the worker tasks for each step:
- call .cancel() so CancelledError is raised. This should stop all IO / http calls (most of the time)
- clear all those 3 queues (except maybe for non-interruptable messages (though I think we should wait before worrying about that) 
- And that's it

I don't see why you need a stop event queue at all. Am I missing something?